skimage/morphology/isotropic.py

```diff
@@ -9,8 +9,9 @@
 def isotropic_erosion(image, radius, out=None, spacing=None):
     """Return binary morphological erosion of an image.
 
-    This function returns the same result as :func:`skimage.morphology.binary_erosion`
-    but performs faster for large circular structuring elements.
+    Compared to the more general :func:`skimage.morphology.erosion`, this
+    function only supports binary input and circular neighborhoods.
```

Comment:
```suggestion
    function only supports binary inputs and circular footprints.
```
It seems that, in the rest of the codebase, we use mostly the term 'footprint' and, sometimes, 'structuring element.'

Reply:
Agreed but I wasn't actually sure if `footprint` would make sense in this case since the function doesn't actually allow passing a `footprint` arg. Instead you define the footprint with a radius. I think "neighborhood" is a term that we are using [elsewhere](https://scikit-image.org/docs/stable/api/skimage.morphology.html#skimage.morphology.closing) and seems to fit better IMO. What do you think?

Reply:
The term 'neighborhood' is more general; in the case of these morphological operations, we talk about 'footprints' or 'structuring elements' (more specifically).

Aww, it seems that the `skimage.morphology.closing` docstring would need some love ('“close” up' instead of '“close up”' and, more importantly, the description for `footprint` isn't great: You may say that the footprint is a kind of neighborhood, but it's weird to start with "THE neighborhood" and then use another term; *and*, let's check, but it shouldn't necessarily be *2D*!!).